src/Murder.Editor/Utilities/OperatingSystemHelpers.cs

```diff
@@ -92,7 +112,11 @@ private static unsafe void Set(void* userdata, byte* text)
 
         if (OperatingSystem.IsMacOS())
         {
-            SetTextForOsx(result);
+            if (HasMacOsAppkit())
+                SetTextForOsx(result);
+            else
+                SDL_SetClipboardText_Mac(result);
```

Comment:
now that we switched to FNA, I think we can just call `SDL2.SDL.SDL_SetClipboardText` and `SDL2.SDL.SDL_GetClipboardText`, instead of manually defining the binding

Reply:
(same for the linux binding, it didn't age well haha)